main.py

```diff
@@ -81,6 +83,28 @@ async def get_experiment_info(file: str) -> Any:
     return remote.examine(file)
 
 
+@app.get("/experiment/submit/")
+async def submit_experiment(file: str, args: str = "{}") -> int:
+    """Submit the given experiment file.
+    
+    Args:
+        file: The path of the experiment file.
+        args: The arguments to submit which must be JSONifiable to a dictionary.
+          Each key is an argument name and its value is the value of the argument.
+    
+    Returns:
+        The run identifier, an integer which is incremented at each experiment submission.
+    """
+    expid = {
+        "log_level": logging.WARNING,
+        "class_name": None,
+        "arguments": json.loads(args),
+        "file": os.path.join(configs["repository_path"], file)
```

Comment:
Let's use the `posixpath` here as well!

Reply:
Good! I will change one of `list_directory()`, too.